### ISSUE TITLE
[Bug]: Sound not playing on server

### ISSUE THREAD

**Issue by violetkitty87** [Mrbysco/SheepSqueak#2]
### General Info

- [X] I am running a modpack
- [ ] I can reproduce this issue consistently in single-player
- [X] I can reproduce this issue consistently in multi-player
- [X] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I am having the same problem.
- [ ] I am crashing and can provide my crash report(s)
- [X] I am using the latest version of the mod

### Loader version

fabric 0.15.11

### Minecraft version

1.21

### Mod version

1.6.0

### Java version

java 8 update 51

### Issue Description

Sheep squeak works on a server but does not produce the noise! i have hit several sheep and no noise. this works just fine on single player.

**Comment by Mrbysco** [comment]
Do the sheep you hurt still have their wool and do you hit with an empty hand? Else it won't play the sound

**Comment by Mrbysco** [comment]
As I'm unable to reproduce this issue on a server

**Comment by violetkitty87** [comment]
Yes they still have their wool its very odd. If you want to test it with the mods I have on the server, I made the pack frisky business blockheads. That pack is the one I'm using that is showing the issue. It may be a conflict with another mod but for some reason it's only in single player that the squeaks work.
